xcpretty.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'rouge', '~> 1.8'
+
```

Comment:
Ok, I think we're on the same page this time, and I may be now convinced this setup is a good thing.

> Currently we use RubyGems, which is bad.
> We should be using Homebrew and treat xcpretty like a CLI tool rather than a library.

Considering there are plenty of other tools already in the average cocoa developer's toolbox which are installed via rubygems (and its on every OS X system already), this still seems negotiable. There are no barriers to installation; rubygems handles libraries with a binary components just fine, just as homebrew, pip, or any other package manager does.

> No dependencies is great because install time is fast, and rubygems are super slow with `dependencies->dependencies->dependencies`.

No argument here, however in this case we are talking about one dependency which greatly enhances the usability and has no dependencies itself. There is the chance that otherwise the rouge gem installed on the system does not meet this requirement and breaks xcpretty as a result when we do implicit dependencies.